src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -252,6 +252,7 @@ namespace InternalHeatGains {
             state.dataHeatBal->ZoneIntGain.allocate(state.dataGlobal->NumOfZones);
             state.dataHeatBal->spaceIntGain.allocate(state.dataGlobal->numSpaces);
             state.dataHeatBal->spaceIntGainDevices.allocate(state.dataGlobal->numSpaces);
+            state.dataDaylightingData->spacePowerReductionFactor.dimension(state.dataGlobal->numSpaces, 1.0);
```

Comment:
Kludgy attempt to get rid of diffs, probably not needed now, but it's all happy now, so . . .